core/src/banking_stage.rs

```diff
@@ -578,12 +580,10 @@ impl BankingStage {
         let mut chunk_start = 0;
         let mut unprocessed_txs = vec![];
         while chunk_start != transactions.len() {
-            let chunk_end = chunk_start
-                + entry::num_will_fit(
-                    &transactions[chunk_start..],
-                    packet::BLOB_DATA_SIZE as u64,
-                    &Entry::serialized_to_blob_size,
-                );
+            let chunk_end = std::cmp::min(
```

Comment:
We deserialize these `transactions` from the packets received from `verified_receiver` in `process_packets()`, do we know how many transactions are coming in from that receiver? If these entries start getting large, the `entries_to_shreds` might get slow in broadcast stage here: https://github.com/solana-labs/solana/blob/master/core/src/broadcast_stage/broadcast_utils.rs#L85. We might want to think about running that entry to shred conversion in separate threads in one of @sagar-solana favorite rayon thread pools